charter.md

```diff
@@ -0,0 +1,34 @@
+#Charter for Working Group
+
+There are many situations in which it is desirable to share a digital credential with another person. For example, you may want to provide access to your vehicle to a Valet. You may also want to provide access to your home to your cat sitter. Or, you may want to share a hotel key with your spouse. Today, no such standardized method exists in a cross-platform, multidisciplinary capacity. 
+
+The WG charter includes the definition and standardization of a protocol that will facilitate such credential transfers from individual to individual. The protocol will leverage a “relay server” to transfer data from sender to recipient. The scope of the transfer is limited to a single origin device and a single destination device.
+
+Privacy goals include:
+
+- The relay server should not see sensitive details of the share
+- The relay server should not be able to provision the credential itself, acting as an intermediary for the recipient (MiTM)
+- The relay server should not know the identity of the sender nor receiver
+- Device OEMs should not be able to identity the social graph of whom has shared with whom
+
+Sufficient security measures should be embedded in the protocol in an effort to:
+
+- Ensure only the intended recipient is able to provision the credential
+- Ensure the credential can only be provisioned once (Anti-replay)
+- Ensure the sender has intent to share (secure user intent)
+- Allow the sender to revoke the credential, either before or after the recipient has accepted, where applicable
```

Comment:
```suggestion
- Allow only the sender and receiver to mutate the sharing channel once a connection has been established.
```

This seems more related to share management and is outside the scope of the IETF spec. I think we could reword this to only cover mailbox deletion.

Reply:
+1 
termination of the credential is specific to the standards it follows, anything outside of the mailbox management should be out of scope. Also, deleting the mailbox may not result in successfully terminating a credential. Example with car keys, deleting the mailbox after sender uploads the key import request will not guarantee terminating the shared car key.

Reply:
+1

Role of relay ends after the sharing flow is over (either with success or failure).

Removing the credential should fall back to lifecycle management  policies of the vertical.